cocos/scene-graph/component-event-handler.ts

```diff
@@ -174,18 +174,18 @@ export class EventHandler {
         handler.apply(comp, params);
     }
 
-    private _compName2Id (compName) {
+    private _compName2Id (compName): any {
         const comp = legacyCC.js.getClassByName(compName);
         return legacyCC.js.getClassId(comp);
     }
 
-    private _compId2Name (compId) {
+    private _compId2Name (compId): any {
```

Comment:
The same as above.

Reply:
here we use legacyCC in this implementation, so TS can only infer the return type as any

Reply:
But we know its return type. Can not change to `string`?

Reply:
then the TS language service would emit and error saying that `Unsafe return of an `any` typed value`